course_catalog/etl/youtube_test.py

```diff
@@ -481,6 +483,8 @@ def test_get_youtube_transcripts_with_a_retry(mocker):
 
     mock_upsert_video = mocker.patch("course_catalog.etl.youtube.upsert_video")
 
+    mocker.patch("time.sleep")
```

Comment:
Instead of mocking, adjust the settings value to a lower value so the code runs but doesn't slow down the tests significantly.